client/src/cli_args.rs

```diff
@@ -251,4 +268,11 @@ impl<'a> EncointerArgsExtractor for ArgMatches<'a> {
 	fn tx_payment_cid_arg(&self) -> Option<&str> {
 		self.value_of(TX_PAYMENT_CID_ARG)
 	}
+	fn at_block_arg(&self) -> Option<Hash> {
+		if let Some(hexhashstr) = self.value_of(AT_BLOCK_ARG) {
+			Some(Hash::from_hex(hexhashstr.to_string()).unwrap())
+		} else {
+			None
+		}
+	}
```

Comment:
This is the verbose version of:

```
self.value_of(AT_BLOCK_ARG)
     .map(|hex| Hash::from(hex.to_string().unwrap())
```

Reply:
haha, thought you might not like these ;-)